### PR TITLE
chore: add new store screen shots for fdroid [WPB-10569]

### PR DESCRIPTION
**Pull request by MohamadJaara**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10569" title="WPB-10569" target="_blank">TaskWPB-10569</a>  [Android] publish fdroid
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

add new store screen shots for fdroid

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/play/release-notes/en-US/default.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Fixed:
+  Team-enforced self-deleting messages are now correctly disabling the button to change it
+  Fixed a bug where the receiver-side of self-deleting messages would fail to delete messages containing files, audios or images.
```
Comment (MohamadJaara) [inline app/src/main/play/release-notes/en-US/default.txt]: waiting for the new change log

File: app/src/main/play/release-notes/en-US/default.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+New
+    Fresh design - completely rebuilt.
+    Ability to add Reactions to messages
+    Swipe left to reply
+    Button to scroll to end of conversation
+    Drafts : keep your message in text field after leaving conversation
```
Comment (mchenani) [inline app/src/main/play/release-notes/en-US/default.txt]:
```suggestion
    Drafts: keep your message in text field after leaving conversation
```

File: app/src/main/play/release-notes/en-US/default.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    Navigate to original message from a reply.
```
Comment (mchenani) [inline app/src/main/play/release-notes/en-US/default.txt]: which is correct? having `.` or not having? it's not constant.

Comment (MohamadJaara) [reply app/src/main/play/release-notes/en-US/default.txt]: I just copied and pasted what product sent me tbh, i will unify all and remove the .

**Review by alexandreferris — APPROVED** [review]

**Pull request merged**